manifoldco/ui
# Dark Theme and Theme Docs

## Reason for change


- Added dark theme
- Modified Gatsby setup so that base theme loads before sample themes.
- A start on theme docs (I think this still needs some work. I'll keep adding to it over time)

## Testing

- Apply Dark theme and ensure components render properly. Also test that other themes still work as expected.
- Go to theming section on docs site and check for accuracy and quality of docs.
